Fix for inverted behaviour of legend horizontal prop

shruthirai opened the pull request.

shadab-skhan reviewed (commented):
Push package-lock.json too

shadab-skhan reviewed (commented):
There are too many diff in package-lock.json
You are using older version of npm hence it created "lockfileVersion": 1 and introduced more changes.
Currently we are using "lockfileVersion": 2, please check

shruthirai commented:
> There are too many diff in package-lock.json You are using older version of npm hence it created "lockfileVersion": 1 and introduced more changes. Currently we are using "lockfileVersion": 2, please check

Updated npm and submitted new lock file. Please check.
